wicket/src/state/inventory.rs

```diff
@@ -138,6 +138,13 @@ fn version_or_unknown(caboose: Option<&SpComponentCaboose>) -> String {
     caboose.map(|c| c.version.as_str()).unwrap_or("UNKNOWN").to_string()
 }
 
+fn caboose_sign(caboose: Option<&SpComponentCaboose>) -> Option<Vec<u8>> {
+    match caboose {
+        None => None,
```

Comment:
Style nit - I think I'd make this function take a non-optional `&SpComponentCaboose`, since we can't do anything useful with `None`. That would also make it clear from the signature that even from a non-optional caboose, we may or may not have a `sign` value.